app/controllers/api/v1/json_web_token.rb

```diff
@@ -0,0 +1,15 @@
+require 'jwt'
+class JsonWebToken
+  extend ActiveSupport::Concern
+  SECRET_KEY = Rails.application.secret_key_base
+
+  def self.jwt_encode(payload, exp = 7.days.from_now)
+    payload[:exp] = exp.to_i
+    JWT.encode(payload, SECRET_KEY)
+  end
+
+  def self.jwt_decode(token)
+    decoded = JWT.decode(token, SECRET_KEY)[0]
```

Comment:
It looks like this have an error because I don't have the secret ket, I invite you to remove this or share it to install the app locally:
<img width="818" alt="image" src="https://user-images.githubusercontent.com/20563252/231896102-f94876fa-45c2-4edc-ac98-471afa6802f0.png">